bpfd/src/main.rs

```diff
@@ -29,6 +29,10 @@ async fn main() -> anyhow::Result<()> {
     setrlimit(Resource::RLIMIT_MEMLOCK, RLIM_INFINITY, RLIM_INFINITY).unwrap();
 
     let config = config_from_file("/etc/bpfd.toml");
-    serve(config, dispatcher_bytes).await?;
+
+    let static_programs: std::collections::HashMap<String, StaticProgram> =
```

Comment:
This isn't what I had in mind. As with most `.d` directories the intention was to have *multiple* config files in this directory. This makes it easier from an admin standpoint (especially when a tool like ansible is in use).
see: https://doc.rust-lang.org/std/fs/fn.read_dir.html

Reply:
Done